src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.blobrouter.services.email;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+
+@SpringBootTest
+@ActiveProfiles("db-test")
```

Comment:
Is the profile correct or needed at all?

Reply:
For some reason it gives an SQLException with 'Could not create new connection' otherwise (played with quite a lot of options).

Reply:
Ah, I recall something, nw then. These setups can be reviewed later